translations/wesnoth-The_Earths_Gut/ja.po

```diff
@@ -2167,8 +2167,8 @@ msgid ""
 "in fact the Hammer of Thursagan, the two of them decided to bring it to "
 "Relgorn."
 msgstr ""
-"Dulatus の取り返したものが 真のThusagan の槌であることを Hamel が確認した後、"
-"二人は Relgorn の許へ持っていくことに決めました。"
+"Dulatus の取り返したものが 真の Thusagan の槌であることを Hamel が確認した"
```

Comment:
oops I merged this before noticing the typo: missing "r" in "Thursagan"

Reply:
Ah..... I'm very sorry.   What can I do?

Reply:
Just be sure to add the missing "r" in your next pull request

Reply:
Are the following steps OK?
If ok, then I will do it immediately.
 step1:  Sync my fork to your master
 step2:  fix forked po file, and pull-req again.